checkrole.yml.sample

```diff
@@ -0,0 +1,30 @@
+---
+
+# check host hardware specification against a given set of Ceph roles.
+#
+# Requires the variables to be passed in by the caller or vars defined
+# e.g.
+# vars:
+#   inventory:
+#     host1: osds,mons
```

Comment:
from what I'm understanding with comments below `osds,mons` would be the groups `host1` would belong to, right ?